### ISSUE TITLE
how was pretrained model built?

### ISSUE THREAD

**Issue by jonmay** [cmusphinx/g2p-seq2seq#77]
What data and parameters were used to build the pretrained model and the model used to generate PRONALSYL 2007 numbers in row "LSTM num_layers=2, size=512" in the table? The table claims 23.3 WER, when I run the pretrained model on the test set of PRONALSYL 2007 I get 22.8, and when I run my own model trained according to the README in the PRONALSYL download 
(--train cmudict.dic.train --test cmudict.dic.test --num_layers 2 --size 512  --max_steps 0) I get 32.11. Is the data the same for all these? Are there any tricks I am missing? Are random restarts needed?

**Comment by nshmyrev** [comment]
You need more steps, something like --max_steps 70000, we are currently working on proper stopping.

**Comment by jonmay** [comment]
ah, thanks, i'll give it a try
